### PR TITLE
Rebase branch 01

### PR DESCRIPTION
**Pull request by MichaelBatsian**
Approve to merge master and rebase-branch-01 branches

### CODE REVIEW

File: db/bears.json [hunk 1]
```diff
@@ -52,12 +52,16 @@
...
       "type": "Grizzly"
-    },
+  },
 	{
 	   "id": 11,
 	   "name": "Bubochka",
 	   "type": "Black"
+	},
+  	{
+	   "id": 12,
```
Comment (kpavaliayeu) [inline db/bears.json]: Merged data is fine. 
See comment about fixing line intend.

File: db/bears.json [hunk 2]
```diff
@@ -52,12 +52,16 @@
       "id": 10,
       "name": "Kenai",
       "type": "Grizzly"
-    },
+  },
```
Comment (kpavaliayeu) [inline db/bears.json]: Line shifted. Please, fix it.
You should check line intend after merging and fix if required.

**Comment by kpavaliayeu** [comment]
Where is a file with text "Lorem ipsum..." from the task #5? Is it exist only locally?

**Comment by MichaelBatsian** [comment]
Line intend fixed, file with text "lorem ..." is stored on the local repository git-example2

**Comment by kpavaliayeu** [comment]
@MichaelBatsian could you please to push local changes with "lorem..." into additional branch?

**Comment by MichaelBatsian** [comment]
Hi, i stored faq.md with text "lorem..." in new branch "rebase-branch-3"

**Pull request merged**